qemu/tests/cfg/rh_qemu_update.cfg

```diff
@@ -0,0 +1,11 @@
+- rh_qemu_update:
+    type = rh_qemu_update
+    only RHEL
```

Comment:
Suggest adding no  `Host_RHEL.m7 Host_RHEL.m8`

Reply:
Good point. I added only `Host_RHEL.m7` (if there is any need to test also RHEL8, test case is ready).